module/VuFind/tests/unit-tests/src/VuFindTest/RecordTab/VersionsTest.php

```diff
@@ -62,10 +62,13 @@ public function testGetDescription(): void
         $obj = new Versions($config, $som);
         $obj->setRecordDriver($recordDriver);
         $translator = $this->getMockBuilder(\Laminas\I18n\Translator\TranslatorInterface::class)
-            ->getMock();
+            ->addMethods(['getLocale'])
```

Comment:
Maybe another place to use the TranslatorTrait? Or is the expectation set below too specific for that to work? (I'm not sure if it really matters that we're expecting the 'default' namespace here... but that's the only major difference I see from using the trait).

Reply:
Using the trait here works. I changed it.